lib/src/main/kotlin/com/swmansion/starknet/account/Account.kt

```diff
@@ -205,24 +205,6 @@ interface Account {
         return signDeployAccountV3(classHash, calldata, salt, params, forFeeEstimate)
     }
 
-    /**
-     * Sign a version 1 declare transaction.
-     *
-     * Prepare and sign a version 1 declare transaction to be executed on Starknet.
-     *
-     * @param contractDefinition a definition of the contract to be declared
-     * @param classHash a class hash of the contract to be declared
-     * @param params additional execution parameters for the transaction
-     * @param forFeeEstimate when set to `true`, it changes the version to `2^128+version` so the signed transaction can only be used for fee estimation
-     * @return signed declare transaction payload
-     */
-    fun signDeclareV1(
-        contractDefinition: Cairo0ContractDefinition,
-        classHash: Felt,
-        params: ExecutionParams,
-        forFeeEstimate: Boolean = false,
-    ): DeclareTransactionV1Payload
```

Comment:
I don't think you removed `DeclareTransactionV1Payload` itself. Let's check for all the code related to Cairo 0 and sending declare v1 and ensure it's all removed.

Reply:
the same applies to https://github.com/software-mansion/starknet-jvm/tree/main/lib/src/test/resources/contracts_v0/src, compile scripts etc etc

Reply:
I've removed Cairo 0 contracts, their compile scripts and related examples from readme files.